src/scss/grommet-core/_objects.card.scss

```diff
@@ -154,4 +154,72 @@ $card-hover-color: #EBEBEB;
       }
     }
   }
+
+  &__label {
+    &.#{$grommet-namespace}heading {
+      letter-spacing: normal;
+      margin-bottom: 0.25em;
+    }
+  }
+
+  &__heading {
+    &.#{$grommet-namespace}heading {
+      margin-bottom: 0.125em;
+    }
+  }
+
+  &__summary {
+    &.#{$grommet-namespace}paragraph {
+      margin-top: 0.125em;
+      margin-bottom: 0.5em;
+    }
+  }
+
+  &__details {
+    &.#{$grommet-namespace}paragraph {
+      margin-bottom: 0.333em;
+    }
+  }
+
+  &__link {
+    margin-top: 0.75em;
+  }
+
+  &--small {
+    .#{$grommet-namespace}card__heading {
+      @include media-query(portable) {
+        font-size: $h4-font-size;
```

Comment:
Do we have styling for `portable` resolution elsewhere? Can we just leave Heading to style the text?